autopush/web/registration.py

```diff
@@ -216,6 +227,26 @@ def _register_channel(self, router_data=None):
         return endpoint, router_data
 
     @threaded_validate(RegistrationSchema)
+    def get(self, *args, **kwargs):
+        """HTTP GET
+
+        Return a list of known channelIDs for a given UAID
+
+        """
+        self.uaid = self.valid_input['uaid']
+        if not self.uaid:
```

Comment:
The validation doesn't actually validate what is required (the UAID in this case)? We added validation schema's to avoid if/else checks in the code here.